server/routes/search.js

```diff
@@ -2,7 +2,7 @@
 
 const router = require('express').Router();
 
-const searchValidator = require('../search/searchValidator');//Use if needed otherwise delete
+
```

Comment:
following your previous comment here: "//Use if needed otherwise delete"
cool to see it in action!

Reply:
twas marks comment =P, but yeahhh i deleted it cause i didnt need it